[2025-04-11T05:12:07.473+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-11T05:05:58.336122+00:00 [queued]>
[2025-04-11T05:12:07.483+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-11T05:05:58.336122+00:00 [queued]>
[2025-04-11T05:12:07.484+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-11T05:12:07.524+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_sqlite> on 2025-04-11 05:05:58.336122+00:00
[2025-04-11T05:12:07.529+0000] {standard_task_runner.py:60} INFO - Started process 203 to run task
[2025-04-11T05:12:07.532+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'load_to_sqlite', 'manual__2025-04-11T05:05:58.336122+00:00', '--job-id', '407', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmp_as3pmnn']
[2025-04-11T05:12:07.534+0000] {standard_task_runner.py:88} INFO - Job 407: Subtask load_to_sqlite
[2025-04-11T05:12:07.605+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-11T05:05:58.336122+00:00 [running]> on host a42cc80ea92f
[2025-04-11T05:12:07.743+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='load_to_sqlite' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T05:05:58.336122+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T05:05:58.336122+00:00'
[2025-04-11T05:12:07.744+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-11T05:12:07.745+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/load_to_sqlite.py']
[2025-04-11T05:12:07.758+0000] {subprocess.py:86} INFO - Output:
[2025-04-11T05:12:08.983+0000] {subprocess.py:93} INFO - âœ… Archivo parquet leÃ­do: /opt/***/data/processed/final_merged.parquet
[2025-04-11T05:12:08.984+0000] {subprocess.py:93} INFO - ðŸ”— Conectado a la base de datos SQLite en /opt/***/data/output/merged_data.db
[2025-04-11T05:12:08.984+0000] {subprocess.py:93} INFO - ðŸ“¦ Datos cargados en la tabla 'datos_finales'
[2025-04-11T05:12:08.984+0000] {subprocess.py:93} INFO - âœ… ConexiÃ³n cerrada. Proceso de carga completado.
[2025-04-11T05:12:09.062+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-11T05:12:09.110+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=load_to_sqlite, execution_date=20250411T050558, start_date=20250411T051207, end_date=20250411T051209
[2025-04-11T05:12:09.188+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-11T05:12:09.209+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
